src/messages/MOSDECSubOpRead.h

```diff
@@ -19,12 +19,12 @@
 #include "osd/ECMsgTypes.h"
 
 class MOSDECSubOpRead : public MOSDFastDispatchOp {
-  static const int HEAD_VERSION = 2;
+  static const int HEAD_VERSION = 3;
   static const int COMPAT_VERSION = 1;
 
 public:
   spg_t pgid;
-  epoch_t map_epoch;
+  epoch_t map_epoch, min_epoch;
```

Comment:
nit, could initialize `min_epoch` to something like 0. yeah, i knew we always initialize it after decoding a message or creating a new one, but it could help us to make sure that the `min_epoch` is initialized properly.